src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageAnalysis.DataFlowAnalyzer.FlowGraphAnalysisData.cs

```diff
@@ -183,7 +183,6 @@ private static PooledDictionary<BasicBlock, BasicBlockAnalysisData> CreateAnalys
             /// <summary>
             /// 
             /// </summary>
```

Comment:
Should this whole block be removed?

Reply:
likely yes :)   just dealing with the simple case for noww: )